src/main/java/com/bieganski/ox/model/Field.java

```diff
@@ -0,0 +1,70 @@
+package com.bieganski.ox.model;
+
+/**
+ * Field represents symbol O or X on given by user position.
+ */
+public class Field implements Comparable<Field> {
```

Comment:
I salute the effort of having docs on public methods here, but I'd like the docs to be present on methods if they add something. Like the bounds case (next comment). If a method is clear-cut like most here, docs are redundant.

Reply:
I think this was resolve by Kamil.